### PR TITLE
Update name

### PR DESCRIPTION
**Pull request by tbwebb22**
This PR removes the name parameter from the initializers for the GovernorModule and the GovernorFactory, since the ModuleBase now has a name parameter. An overriding name() function is also added to the GovernorModule.

### CODE REVIEW

File: contracts/Governor/GovernorFactory.sol [hunk 1]
```diff
@@ -64,14 +64,13 @@ contract GovernorFactory is ERC165, ModuleFactoryBase {
...
                     abi.decode(data[2], (address)),
                     timelock,
-                    abi.decode(data[6], (uint64)),
+                    abi.decode(data[5], (uint64)),
+                    abi.decode(data[6], (uint256)),
                     abi.decode(data[7], (uint256)),
                     abi.decode(data[8], (uint256)),
                     abi.decode(data[9], (uint256)),
-                    abi.decode(data[10], (uint256)),
                     abi.decode(data[1], (address))
```
Comment (adamgall) [inline contracts/Governor/GovernorFactory.sol:74]: Do you mind commenting on what these changes are?

File: contracts/Governor/GovernorModule.sol [hunk 2]
```diff
@@ -42,7 +40,7 @@ contract GovernorModule is
         uint256 _initialQuorumNumeratorValue,
         address _accessControl
     ) external initializer {
-        __Governor_init(_name);
+        __Governor_init("GovernorModule");
```
Comment (adamgall) [inline contracts/Governor/GovernorModule.sol]: Should this have a space in it? idk

**Review by adamgall — COMMENTED** [review]
Just some comments, not necessarily holding up the review

**Review by adamgall — APPROVED** [review]

File: contracts/Governor/GovernorModule.sol (hunk 2 above)
Comment (tbwebb22) [reply contracts/Governor/GovernorModule.sol]: Added space

File: contracts/Governor/GovernorFactory.sol (hunk 1 above)
Comment (tbwebb22) [reply contracts/Governor/GovernorFactory.sol:74]: Yes, so the name was being passed in the data[5] index. That got removed, so all of the array indexes above 5 needed to be shifted down to fill in that empty array index.

**Pull request merged**